src/app/overview/home/home.component.html

```diff
@@ -65,28 +77,10 @@
 
   <ng-container *ngIf="todoItems.length; else noItemsTpl">
     <div *ngFor="let todoItem of todoItems" @newLoaded>
-      <ng-container [ngSwitch]="todoItem.type">
-        <app-todo-card
-          *ngSwitchCase="'feedback_available'"
-          [todoItem]="todoItem"
-          (click)="goToAssessment(todoItem.meta.activity_id, todoItem.meta.context_id, todoItem.meta.assessment_id)"
-        ></app-todo-card>
-        <app-todo-card
-          *ngSwitchCase="'review_submission'"
-          [todoItem]="todoItem"
-          (click)="goToReview(todoItem.meta.context_id, todoItem.meta.assessment_id, todoItem.meta.assessment_submission_id)"
-        ></app-todo-card>
-        <app-todo-card
-          *ngSwitchCase="'chat'"
-          [todoItem]="todoItem"
-          (click)="goToChat(todoItem)"
-        ></app-todo-card>
-        <app-todo-card
-          *ngSwitchCase="'assessment_submission_reminder'"
-          [todoItem]="todoItem"
-          (click)="goToAssessment(todoItem.meta.activity_id, todoItem.meta.context_id, todoItem.meta.assessment_id)"
-        ></app-todo-card>
-      </ng-container>
+      <app-todo-card
+        [todoItem]="todoItem"
+        (click)="goToAssessments(todoItem)"
```

Comment:
better to change the name of this function to a more generic name
maybe something like "clickTodoItem"